presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -290,6 +290,12 @@ std::string SystemConfig::prestoVersion() const {
   return requiredProperty(std::string(kPrestoVersion));
 }
 
+std::string SystemConfig::poolType() const {
```

Comment:
Wanted to confirm that this is not a registered property... Else it has to be a part of registeredProps_. If you add it to registeredProps_ it might be simpler to initialize the default value as well.

Reply:
What is the difference between registeredProps_ and configs?

Reply:
Registered properties are validated on server startup https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/common/Configs.cpp#L110

Might be better to add it to avoid warnings.